hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -669,6 +677,259 @@
         ]
       }
     }
+    //===============================================================
+    // TPM registers
+    { skipto: "0x800"}
+    { name: "TPM_CAP"
+      desc: '''TPM HWIP Capability register.
+
+        This register shows the features the current TPM HWIP supports.
+        '''
+      swaccess: "ro"
+      hwaccess: "hwo"
+      fields: [
+        { bits: "7:0"
+          name: "rev"
+          desc: "Revision of the TPM submodule"
+        } // f: rev
+        { bits: "8"
+          name: "locality"
+          desc: '''If 1, the TPM submodule supports 5 Locality.
+            If 0, only one Locality is provided
+            '''
+        } // f: locality
+        { bits: "18:16"
+          name: "max_xfer_size"
+          desc: '''The maximum transfer size in bytes the TPM submodule supports.
+            The value is the exponent of the 2.
+
+            - 3'b 010: Support up to 4B
+            - 3'b 011: Support up to 8B
+            - 3'b 101: Support up to 32B
+            - 3'b 110: Support up to 64B
+
+            All other values are reserved.
+
+            The SW may advertises as supporting more than `max_xfer_size` to the South Brige.
+            If the SW reports the transfer size more than the value described in this field,
+            the SW must read the data from the write FIFO before the FIFO becomes full in order not to make FIFO overflow.
+            '''
+        } // f: max_xfer_size
+      ]
+      tags: [
+        // Remove Read check (reset value)
+        "excl:CsrAllTests:CsrExclCheck"]
+    } // R: TPM_CAP
+    { name: "TPM_CFG"
+      desc: '''TPM Configuration register.
+        '''
+      swaccess: "rw"
+      hwaccess: "hro"
+      fields: [
+        { bits: "0"
+          name: "en"
+          desc: "If 1, TPM submodule accepts the transactions over SPI"
+        } // f: en
+        { bits: "1"
+          name: "hw_reg_dis"
+          desc: '''If 0, TPM submodule directly returns the return-by-HW registers for the read requests.
+
+            If 1, TPM submodule uploads the TPM command regardless of the address, and the SW may return the value through the read FIFO.
+            '''
+        } // f: hw_reg_dis
+        { bits: "2"
+          name: "invalid_ool"
+          desc: '''If 1, TPM submodule returns the invalid data (0xFF) for the out of the max Locality request.
```

Comment:
i think maybe i would say "unmapped registers", assuming the locality refers to what i think it is.

Reply:
o wait, Locality is a term in the spec. Unmapped Locality..?

Reply:
I really couldn't come up with appropriate term here :)